Change broader term classifier to a classification scheme classifier

jonescc opened the pull request:
which creates category paths to be indexed for facetted searching by looking up the parent categories for terms defined in a vocabulary thesaurus
in a classification scheme thesaurus

jonescc commented:
Requires changes to chef to support configuration of vocabulary and classification thesauri to use to index parameter and platform facets and the modified config-summary.xml configuration

danfruehauf commented on web/src/main/java/org/fao/geonet/kernel/search/classifier/ClassificationScheme.java:
```diff
@@ -0,0 +1,129 @@
+//=============================================================================
```
Is that an imported file?

jonescc replied:
Not sure what you are referring to here web/src/main/java/org/fao/geonet/kernel/search/classifier/ClassificationScheme.java or the Geonetwork copyright notice.

danfruehauf replied:
The file itself, and whether it should have this copyright.

jonescc replied:
Well I wrote the class so perhaps not.

jonescc replied:
Followed this up with @pblain who wlll check the requirement here.  GeoNetwork have specific requirements re licensing for contributed code.

danfruehauf commented on web/src/test/java/org/fao/geonet/test/TestThesaurusManager.java:
```diff
@@ -0,0 +1,76 @@
+package org.fao.geonet.test;
```
I think the filename should be `ThesaurusManagerTest.java`, no?

jonescc replied:
Well.  Its not a test - its a thesaurusmanger for use by tests.

jonescc replied:
Perhaps DirectoryThesaurusLoader would be better

jonescc commented:
Removed FAO license and renamed TestThesaurusManager to ThesaurusDirectoryLoader.  Also reverted delete of BroaderTerm classifier - not sure why I deleted it in the first place and it simplifies migration
